core/account-id/src/errors.rs

```diff
@@ -1,14 +1,16 @@
 use std::fmt;
 
 /// An error occurred when parsing an invalid Account ID with [`AccountId::validate`](crate::AccountId::validate).
```

Comment:
This reads a bit awkward to me: this is data, not action, and `validate` seems overly specific. I'd just steal from the great https://doc.rust-lang.org/stable/std/num/struct.ParseIntError.html

```
An error which can be returned when parsing a NEAR account ID.
```